news/2017-08-04-scorewatch-july-week-4.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Scorewatch: July Week 4"
+permalink: http://osunews.tumblr.com/post/162432090863/scorewatch-july-week-4
+date: 2017-08-04 15:00:00 +0200
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-july-week-4
+---
+July has finally ended but it definitely did not end without a bang! An amazing EZDTFL play on a popular map, a couple of number 1 plays and more crazy non osu!standard scores, all you do not want to miss!
```

Comment:
L8: Suggestion: probably short hand do not want to don't ``[..] all you do not want to miss`` > ``[..] all you don't want to miss``